packages/block-library/src/nextpage/edit.native.js

```diff
@@ -14,20 +14,21 @@ import { __ } from '@wordpress/i18n';
  */
 import styles from './editor.scss';
 
-export default function NextPageEdit( { attributes } ) {
-	const { customText = __( 'Page break' ) } = attributes;
-	// Setting the font here to keep the CSS linter happy, it was demanding a syntax
-	// that React Native wasn't able to handle (adding a fallback generic font family).
-	const textStyle = {
-		...styles[ 'block-library-nextpage__text' ],
-		fontFamily: 'System',
```

Comment:
This was creating an issue that disabled scrolling with VoiceOver active.
I'm not sure the specific reason, but rolling back this change fixed the issue.